### PR TITLE
libxkbcommon 0.7.1

### PR DESCRIPTION
**Pull request by ilovezfs**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by apjanke** [comment]
Looks like the `configure` script has changed; our `inreplace` is failing.

```
tar xJf /Users/brew/Library/Caches/Homebrew/libxkbcommon-0.7.0.tar.xz
Error: ["configure", ["expected replacement of \"${wl}-version-script\" with \"\""]]
```

**Comment by JCount** [comment]
@ilovezfs I have tinkered and put together a working version of this. I can give you the diff if you would like. (I fixed the head spec while I was at it too.)

File: libxkbcommon.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class Libxkbcommon < Formula
   desc "Keyboard handling library"
   homepage "http://xkbcommon.org/"
-  url "http://xkbcommon.org/download/libxkbcommon-0.5.0.tar.xz"
-  sha256 "90bd7824742b9a6f52a6cf80e2cadd6f5349cf600a358d08260772615b89d19c"
+  url "http://xkbcommon.org/download/libxkbcommon-0.7.0.tar.xz"
```
Comment (vszakats) [inline libxkbcommon.rb]: Can you please bump the protocol to HTTPS?

File: libxkbcommon.rb [hunk 2]
```diff
@@ -1,8 +1,8 @@
 class Libxkbcommon < Formula
   desc "Keyboard handling library"
   homepage "http://xkbcommon.org/"
```
Comment (vszakats) [inline libxkbcommon.rb]: Can you please bump the protocol to HTTPS? (#1)

**Comment by JCount** [comment]
closing in favor of #290

**Pull request closed without merging**